tlib/src/system.cpp

```diff
@@ -99,7 +99,8 @@ std::expected<size_t> tlib::exec_and_wait(const char* executable, const std::vec
     return std::move(result);
 }
 
-void tlib::reboot(){
+void tlib::reboot(int delay=0) {
+    tlib::sleep_ms(1000*delay);
```

Comment:
Could you add a test to do the sleep only if delay is not zero? And add spaces around *.